tests/test_vfolder.py

```diff
@@ -104,16 +104,19 @@ def test_vfolder_get_info():
 
 
 def test_vfolder_upload(tmp_path: Path):
```

Comment:
Could you rewrite the test to just check the JWT token issued upon the upload request?

Reply:
I have multiple problems with utilizing the test.

In original upload test the function sends request:
m.post(build_url(session.config,
                         '/folders/{}/upload'.format(vfolder_name)),
               status=201)

1. Problem. Now with tus upload the message should be signed.
2. Vfolder upload command doesn't work if aioresponses() included in context manager. The server rejects the request.
3.  The tus server cannot see the tmp_path directory which it comes as argument. And it also can't find the directory if tmp_path = Path.cwd() in test directory

Reply:
JWT token test is complete